twas-nd/src/main/resources/marketing-artifacts/partner-center.html

```diff
@@ -1,29 +1,29 @@
 <h1>Azure Virtual Machine Offer</h1>
 <h2>Offer setup</h2>
 <h3>Alias</h3>
-<p>IBM WebSphere Application Server ND 9.0.5.x Image and JDK 8 on RHEL 9.x</p>
+<p>IBM WebSphere Application Server Network Deployment 9.0.5.x Image</p>
 <h2>Offer listing</h2>
 <h3>Name</h3>
-<p>IBM WebSphere Application Server ND 9.0.5.x Image and JDK 8 on RHEL 9.x</p>
+<p>IBM WebSphere Application Server Network Deployment 9.0.5.x Image</p>
 <h3>Search results summary</h3>
-<p>IBM WebSphere Application Server ND 9.0.5.x Image and JDK 8 on Red Hat Enterprise Linux 9.x</p>
+<p>IBM WebSphere Application Server Network Deployment 9.0.5.x Image</p>
 <h3>Short description</h3>
-<p>IBM WebSphere Application Server ND 9.0.5.x Image and JDK 8 on Red Hat Enterprise Linux 9.x.</p>
+<p>This image includes the latest IBM WebSphere Application Server ND using IBM JDK 8 on RHEL 9.x</p>
 <h3>Description</h3>
-<p>This is an IBM WebSphere Application Server Network Deployment 9.0.5.x Image and JDK 8 with Red Hat Enterprise Linux 9.x.</p>
-<p>The VM instance created by this offer is suitable for customers that require very highly customized Azure deployments. 
-Users of this offer are expected to be familiar with WebSphere Application Server administration. The VM is installed with
-the specified GNU/Linux distribution, IBM JDK version, and WebSphere Application Server Network Deployment version. 
-The relevant directories are listed here:</p>
+<p>This offer is an IBM WebSphere Application Server Network Deployment 9.0.5.x Image and JDK 8 with Red Hat Enterprise Linux 9.x.</p>
+<p>The VM instance that this offer creates is suitable for customers who require customizable Azure deployments with a configurable 
+    WebSphere Application Server deployment. Users of this offer are expected to be familiar with WebSphere Application Server administration. 
+    The VM is installed with the specified GNU/Linux distribution, IBM JDK version, and WebSphere Application Server Network Deployment version. 
+    The relevant directories are as follows:</p>
 <ul>
 <li>WAS_INSTALL_ROOT: /datadrive/IBM/WebSphere/ND/V9</li>
 <li>JAVA_HOME: ${WAS_INSTALL_ROOT}/java/8.0</li>
 </ul>
-<p>There is no pre-created cluster. During VM creation, you are asked to provide a username and SSH key or credentials for the admin account.
-After logging in with this account, do sudo -i to get write access to the relevant directories.</p>
-<p>In addition to the VM ND image, other offers are available to meet different needs.  These includes solution templates for 
+<p>Clusters, profiles, and server instances are not pre-created. During VM creation, you are asked to provide a username and SSH key or credentials 
+    for the admin account. After you log in with this account, do sudo -i to get write access to the relevant directories.</p>
```

Comment:
Update "sudo -i" to "\<code>sudo -i\</code>", which is consistent with other occurrences.

Reply:
Hi @majguo while it is important to be consistent, the partner center UI for Azure Virtual Machines offer listing feature does not support HTML. Only plain text. So any formatting in the HTML is simply lost anyway.

Reply:
@edburns I am planning to use HTML tags to resolve newline issue. According [Azure Offer listing docs](https://learn.microsoft.com/en-us/partner-center/marketplace/create-new-saas-offer-listing) the description box supports HTML. I hope it will work.